src/com/google/javascript/jscomp/js/es6/util/inherits.js

```diff
@@ -58,14 +58,17 @@ $jscomp.inherits = function(childCtor, parentCtor) {
   /** @override */
     childCtor.prototype.constructor = childCtor;
 
-  for (var p in parentCtor) {
-    if (Object.defineProperties) {
-      var descriptor = Object.getOwnPropertyDescriptor(parentCtor, p);
-      if (descriptor) {
-        Object.defineProperty(childCtor, p, descriptor);
+  if (Object.defineProperties) {
+    var propNames = Object.getOwnPropertyNames(parentCtor);
+    for (var i = 0; i < propNames.length; i++) {
+      var descriptor = Object.getOwnPropertyDescriptor(parentCtor, propNames[i]);
+      if (descriptor && (descriptor.enumerable || descriptor.configurable))) {
```

Comment:
I believe travis is failing, because there's one to many closing parentheses on this line.